ONSdigital/ras-frontstage
# Fix ras frontstage package security issue

# Motivation and Context
There are multiple vulnerable packages.  This PR fixes them, but pins redis to 2.X as the upgrade to 3.X breaks some functionality.

# What has changed
Nothing that a user can see.  Various packages have been updated and redis has been pinned to ensure it keeps on working.

# How to test?
Unit tests should still pass.
Acceptance tests should pass (on my run, 1 failed but it looked unrelated when I tested the functionality).  I had to check out the tag for frontstage as one of the tests fell over when I had a local copy running (as opposed to a docker container)
Use frontstage yourself, it should still all work as expected.

# Links
https://trello.com/c/swUJ0rJf/810-fix-ras-frontstage-package-security-issue-2

# Screenshots (if appropriate):
